### PR TITLE
Implement placemark level of details selector

### LINKED ISSUE #34: Placemark level of details selection is missing
WorldWindJS implementation of Placemark is missing level of details selection for placemark image source, depending on camera distance, as it was already done in Java and Android implementations.

Function makeOrderedRenderable in Placemark should execute some selectLod(dc, this, this.eyeDistance) function before this.determineActiveAttributes(dc) to allow application to define placemark attributes selection depending on this.eyeDistance.

You can look at example in Android repository.

### PR DESCRIPTION
**Pull request by EMaksymenko**
### Description of the Change
Implement Placemark.selectLevelOfDetail function, which allows to override placemark attributes based on camera distance, highlight status etc.

### Why Should This Be In Core?
This feature is present in Java and Android SDK, but missing in JS. It is suitable for advanced placemark visualization management.

### Benefits
This enhancement allows e.g. to create different tactical graphics placemark visualization based on camera distance.

### Potential Drawbacks
None

### Applicable Issues
Closes #34

### CODE REVIEW

File: src/shapes/Placemark.js [hunk 1]
```diff
@@ -502,6 +504,17 @@ define([
             return this;
         };
 
+        /**
+         * Gets the active attributes for the current distance to the camera and highlighted state.
```
Comment (emxsys) [inline src/shapes/Placemark.js]:
```suggestion
         * Set the active attributes for the current distance to the camera and highlighted state.
```

Comment (emxsys) [inline src/shapes/Placemark.js]:
```suggestion
         * Set the active attributes for the current distance to the camera and highlighted state.
```

**Review by emxsys — COMMENTED** [review]
Thanks @Sufaev for completing this PR.  I made one minor suggestion to improve the documentation. 

I will investigate why Travis-CI is failing and then approve this PR.

**Comment by EMaksymenko** [comment]
I have copied this comment from Android source.
By the way it is completely wrong in Android also, because indeed it sets not active but actual (normal) attributes inside. Active attributes are defined in the next function after level of details.

Correct comment probably should be "Set the placemark attributes..."

File: src/shapes/Placemark.js (hunk 1 above)
Comment (emxsys) [inline src/shapes/Placemark.js]:
```suggestion
         * Set the placemark attributes for the current distance to the camera and highlighted state.
```

Comment (emxsys) [reply src/shapes/Placemark.js]: @Sufaev I can't commit the comment suggestion in your repo.  If you can make the commit, then I'll merge the PR.

**Comment by EMaksymenko** [comment]
Done.

**Pull request merged**